src/tokens.ts

```diff
@@ -233,13 +242,20 @@ export function tokensFor(d: Dialect) {
       input.acceptToken(BracketR)
     } else if (next == Ch.Semi) {
       input.acceptToken(Semi)
-    } else if (next == Ch._0 && (input.next == Ch.b || input.next == Ch.B) ||
-               (next == Ch.b || next == Ch.B) && input.next == Ch.SingleQuote) {
-      let quoted = input.next == Ch.SingleQuote
+    } else if (d.unquotedBitLiterals && next == Ch._0 && input.next == Ch.b) {
```

Comment:
It looks like the `0b` format was parsed by default before and now requires an option to be enabled. Is that intentional?

Same question for no longer accepting `0B`.

Reply:
You are correct, I have found 0b to be supported in MySQL only, hence only enabling that route for those parsers since PostgreSQL for example would parse it as a integer and then get a syntax error at `b`. But I’m open to switch the default if we want to be backwards-compatible.

0B is removed because it’s not valid according to MySQL docs.